src/main/kotlin/org/jitsi/nlj/RtpSenderImpl.kt

```diff
@@ -86,6 +87,7 @@ class RtpSenderImpl(
 
     private val srtpEncryptWrapper = SrtpTransformerNode("SRTP encrypt")
     private val srtcpEncryptWrapper = SrtpTransformerNode("SRTCP encrypt")
+    private val toggleablePcapWriter = ToggleablePcapWriter(logger, id)
```

Comment:
Perhaps use something like `$id-sender` as the prefix (and similar in `RtpReceiverImpl`), to make it easier to distinguish the two files when both egress and ingress dumping has been enabled for an endpoint?

Reply:
I suppose another change here is that we've not got the data in 2 files, instead of 1.  To get back to one we'd have to create the pcapwriter in transceiver and pass it in or something.

Reply:
> To get back to one we'd have to create the pcapwriter in transceiver and pass it in or something.

https://github.com/jitsi/jitsi-media-transform/pull/169/commits/d3f41bc8747287491dc0bfec2ae3adad675c68d0 attempts to do that

Reply:
(we have one pcapwriter per transceiver and 4 observer nodes, two for rtp/rtcp audio and two for rtp/rtcp video)

Reply:
I'm wondering if we want to give the ingress and egress different (fake) IP addresses or ports, to make it easier to disambiguate them in the capture?

Reply:
@gpolitis we had some discussion in the channel (feel free to add thoughts there), but we thought we should leave it as 2 separate pcap files (each named appropriately to denote TX or RX).  This way it avoids some of the awkwardness of having to pass things  in when instantiating senders/receivers and it alleviates any thread safety concerns.

Reply:
ah, yes, that makes sense. I'll revert my latest commit and adjust the code accordingly.

Reply:
done